### PR TITLE
Minor: remove compat from cookbook

### LINKED ISSUE #184: Update CSS Cookbook
### Problem statement

The Cookbook section is outdated, with indicators that well supported features are not fully supported yet.

### Proposed solutions

Update ALL. THE. THINGS.

### Task list

Upated the following pages and removed browser compact (except baseline, if present)

- [X] [Media objects](/en-US/docs/Web/CSS/Layout_cookbook/Media_objects)   https://github.com/mdn/content/pull/30142
- [x] [Columns](/en-US/docs/Web/CSS/Layout_cookbook/Column_layouts) https://github.com/mdn/content/pull/29961
- [x] [Center an element](/en-US/docs/Web/CSS/Layout_cookbook/Center_an_element) (https://github.com/mdn/content/pull/29609)
- [X] [Sticky footers](/en-US/docs/Web/CSS/Layout_cookbook/Sticky_footers)  https://github.com/mdn/content/pull/30146
- [X]  [Split navigation](/en-US/docs/Web/CSS/Layout_cookbook/Split_Navigation)   https://github.com/mdn/content/pull/30145
- [x]  [Breadcrumb navigation](/en-US/docs/Web/CSS/Layout_cookbook/Breadcrumb_Navigation)  (https://github.com/mdn/content/pull/29921)
- [x]  [List group with badges](/en-US/docs/Web/CSS/Layout_cookbook/List_group_with_badges) 
- [X] [Pagination](/en-US/docs/Web/CSS/Layout_cookbook/Pagination)  https://github.com/mdn/content/pull/30143/
- [X] [Card](/en-US/docs/Web/CSS/Layout_cookbook/Card) https://github.com/mdn/content/pull/29972
- [x] [Grid wrapper](/en-US/docs/Web/CSS/Layout_cookbook/Grid_wrapper)  https://github.com/mdn/content/pull/29998  

Also, 
- [x] [Contribute a recipe](/en-US/docs/Web/CSS/Layout_cookbook/Contribute_a_recipe)   https://github.com/mdn/content/pull/29993

### PR DESCRIPTION
**Pull request by estelle**
Update the content to omit any "lack of support"

fixes https://github.com/openwebdocs/project/issues/184.

### CODE REVIEW

File: files/en-us/web/css/layout_cookbook/list_group_with_badges/index.md [hunk 1]
```diff
@@ -15,7 +13,7 @@ In this recipe we will create a list group pattern with badges that indicate a c
 
 ## Requirements
 
-Our list items should be displayed with the badges lined up on the right, no matter how much content the item has. The badge should always be centered vertically whether there is a single line of content, or more than one.
+The list items should be displayed with the badges. The badge should be aligned right and vertically centered. The badge should must be centered vertically whether there is a single line of content or multiple lines of text.
```
Comment (chrisdavidmills) [inline files/en-us/web/css/layout_cookbook/list_group_with_badges/index.md:16]:
```suggestion
The list items should be displayed with the badges. The badge should be aligned right and vertically centered. The badge must be centered vertically whether there is a single line of content or multiple lines of text.
```

File: files/en-us/web/css/layout_cookbook/list_group_with_badges/index.md [hunk 2]
```diff
@@ -27,17 +25,14 @@ Our list items should be displayed with the badges lined up on the right, no mat
 
 ## Choices made
 
-Flexbox makes this particular pattern straightforward and also makes it easy to make changes to the layout.
+Flexbox makes this particular pattern straightforward and also enables making changes to the layout.
```
Comment (chrisdavidmills) [inline files/en-us/web/css/layout_cookbook/list_group_with_badges/index.md:28]:
```suggestion
Flexbox makes this particular pattern straightforward and also facilitates making changes to the layout.
```

Given that the point is that it makes it easier, not just enables it full stop?

**Review by chrisdavidmills — CHANGES_REQUESTED** [review]
@estelle Nice job! Couple of very small bits.

**Review by teoli2003 — APPROVED** [review]
👍

File: files/en-us/web/css/layout_cookbook/list_group_with_badges/index.md [hunk 3]
```diff
@@ -27,17 +25,14 @@ Our list items should be displayed with the badges lined up on the right, no mat
...
-## Browser compatibility
-
-{{Compat}}
+To align the content horizontally, I use the {{cssxref("align-items")}} property to align the text and badge on the cross axis. If you want the badge to align to the top of the content instead, change this to `align-items: flex-start`.
 
 ## See also
 
 - [Box alignment in Flexbox](/en-US/docs/Web/CSS/CSS_box_alignment/Box_alignment_in_flexbox)
 - [Aligning items in a flex container](/en-US/docs/Web/CSS/CSS_flexible_box_layout/Aligning_items_in_a_flex_container)
+- [CSS flexible box layout](/en-US/docs/Web/CSS/CSS_flexible_box_layout/) module
```
Comment (mathewhodson) [inline files/en-us/web/css/layout_cookbook/list_group_with_badges/index.md:38]: The bot is reporting a flaw with this link. Remove the trailing slash.

File: files/en-us/web/css/layout_cookbook/sticky_footers/index.md [hunk 4]
```diff
@@ -40,16 +36,16 @@ Grid auto-placement will place our items in source order and so the header goes
...
-## Browser compatibility
-
-{{Compat}}
-
 ## Resources on MDN
 
-- [Basic concepts of Grid Layout](/en-US/docs/Web/CSS/CSS_grid_layout/Basic_concepts_of_grid_layout)
+- CSS properties: {{cssxref("display")}}, {{cssxref("min-height")}}, {{cssxref("grid-template-rows")}}, {{cssxref("flex-direction")}}, {{cssxref("flex-grow")}}, {{cssxref("flex-shrink")}}
+- [Basic concepts of grid layout](/en-US/docs/Web/CSS/CSS_grid_layout/Basic_concepts_of_grid_layout)
+- [CSS grid layout](/en-US/docs/Web/CSS/CSS_grid_layout/) module
```
Comment (mathewhodson) [inline files/en-us/web/css/layout_cookbook/sticky_footers/index.md:49]: Same here. Need to remove the trailing slash to fix the flaw.

File: files/en-us/web/css/layout_cookbook/list_group_with_badges/index.md (hunk 3 above)
Comment (estelle) [inline files/en-us/web/css/layout_cookbook/list_group_with_badges/index.md:38]:
```suggestion
- [CSS flexible box layout](/en-US/docs/Web/CSS/CSS_flexible_box_layout) module
```

File: files/en-us/web/css/layout_cookbook/sticky_footers/index.md (hunk 4 above)
Comment (estelle) [inline files/en-us/web/css/layout_cookbook/sticky_footers/index.md:49]:
```suggestion
- [CSS grid layout](/en-US/docs/Web/CSS/CSS_grid_layout) module
```

**Comment by teoli2003** [comment]
pinging @chrisdavidmills here

**Review by chrisdavidmills — APPROVED** [review]
This is ready to merge; thanks all!

**Pull request merged**